src/services/campaign-delivery.js

```diff
@@ -34,8 +34,8 @@ module.exports = {
   },
 
   parseLimit(num) {
-    const limit = num > 0 ? parseInt(num, 10) : 1;
-    if (limit > 10) throw createError(400, 'You cannot return more than 10 ads in one request.');
+    const limit = num > 0 ? parseInt(num, 50) : 1;
```

Comment:
@Shinsina This line shouldn't need to change.  The only thing that should change here to increase the max limit is the next line.  `if (limit > 50)`

I believe parseInt(num, 50) will return  NaN no matter what you set num.